app/views/content/get-school-experience/get-school-experience.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Get school experience
+image: "/assets/images/international-dt.jpg"
+mobileimage: "/assets/images/steps-hero-mob.jpg"
+description: |-
+  School experience can help your teacher training application. Here's how to get school experience.
+date: "2021-05-04"
+right_column:
+  ctas:
+    - title: Find school experience
+      text: |
+       Request early years, primary or secondary experience.
+      link_text: "Find school experience"
+      link_target: "https://schoolexperience.education.gov.uk/"
+      icon: "icon-person"
+      hide_on_mobile: Yes
+      hide_on_tablet: Yes
+keywords:
+  - School experience
+  - Teaching experience
+  - Teacher experience
+---
+
+School experience can help you decide whether teaching is right for you.
+
+Spending some time in a school allows you to:
+
+* observe lessons and other teacher responsibilities
+* talk to teachers
+* get an insight into day-to-day school life
+
+It can also really help [your teacher training application](/tips-on-applying-for-teacher-training), although it’s not a training entry requirement.
+
+To find experience you can:
+
+* [request early years, primary or secondary experience on GOV.UK](https://schoolexperience.education.gov.uk/) - not all schools advertise experience here and some only offer online experience due to coronavirus (COVID-19) restrictions
+* approach schools yourself
+
+## Approach schools yourself
+
+[Find schools near you](https://get-information-schools.service.gov.uk/) and give them a ring to find out who to talk to about getting school experience.
+
+It's normal to have to try a few schools before you find one that can help.
+
+They're not obliged to offer experience and may be limiting visitors due to coronavirus (COVID-19).
+
+They also need to be careful about allowing visitors into the school for the safety of children and young people.
+
+You may also be asked to have a DBS check to ensure you're safe to work with children.
+
+Be flexible with your availability if possible and persevere with your search.
+
+## Maths or physics teaching internships
+
+[Find a paid teaching internship if you’re doing a STEM undergraduate degree](https://getintoteaching.education.gov.uk/teaching-internship-providers).
+
+## If you cannot get into the classroom right now
+
+[Observe teachers’ lessons and analysis of their lessons on the Oak National Academy website](https://teachers.thenational.academy/lessons-for-itt).
```

Comment:
This page looks a bit broken. Can we link to a YouTube channel or something instead?

Reply:
Couldn't agree more - I think it's the specific videos on this page though that we've been asked to link to... :/

Reply:
Yep - this is a policy steer on where we should link to